### PR TITLE
CEDS-2387 - fix for submissions with pseudo status 'Pending'

### PR DESCRIPTION
**Pull request by SteveSugden**
These are currently being displayed in all three tabs - Rejected, Actions Required and Other
They should only be in Other until their real status is determined.

### CODE REVIEW

File: test/controllers/util/SubmissionDisplayHelperSpec.scala [hunk 1]
```diff
@@ -166,18 +166,25 @@ class SubmissionDisplayHelperSpec extends WordSpec with MustMatchers {
...
+        val submissions = Seq(submission -> Seq(notification), submission_3 -> Seq(notification_2), submission_3 -> Seq.empty)
+
+        val results = SubmissionDisplayHelper.filterSubmissions(
+          submissions,
+          notifications => notifications.isEmpty || notifications.headOption.map(_.status).exists(SubmissionStatus.otherStatuses.contains)
+        )
 
-        SubmissionDisplayHelper.filterSubmissions(submissions, _ => true) must equal(submissions)
-        SubmissionDisplayHelper.filterSubmissions(submissions, _ => false) must equal(submissions)
+        results must equal(Seq(submission -> Seq(notification), Seq(submission_3 -> Seq.empty)))
```
Comment (MaciejRewera) [inline test/controllers/util/SubmissionDisplayHelperSpec.scala:187]: Something does not seem right here... the first element is of type `Submission -> Seq[Notification]`, but the second one is `Seq[Submission -> Seq[Notification]]`. Should the second element be wrapped in `Seq`?

Comment (SteveSugden) [reply test/controllers/util/SubmissionDisplayHelperSpec.scala:187]: Yep.  Test fails.  But well spotted.

Comment (SteveSugden) [reply test/controllers/util/SubmissionDisplayHelperSpec.scala]: We can turn off PR build and rely on Maciej code reviews!  :-)

File: test/controllers/util/SubmissionDisplayHelperSpec.scala [hunk 2]
```diff
@@ -166,18 +166,25 @@ class SubmissionDisplayHelperSpec extends WordSpec with MustMatchers {
...
+        val submissions = Seq(submission -> Seq(notification), submission_2 -> Seq(notification_2), submission_3 -> Seq.empty)
+
+        val results = SubmissionDisplayHelper.filterSubmissions(
+          submissions,
+          notifications => notifications.isEmpty || notifications.headOption.map(_.status).exists(SubmissionStatus.otherStatuses.contains)
+        )
 
-        SubmissionDisplayHelper.filterSubmissions(submissions, _ => true) must equal(submissions)
-        SubmissionDisplayHelper.filterSubmissions(submissions, _ => false) must equal(submissions)
+        results must equal(Seq(submission -> Seq(notification), submission_3 -> Seq.empty))
```
Comment (MaciejRewera) [inline test/controllers/util/SubmissionDisplayHelperSpec.scala:187]: 👍

**Comment by SteveSugden** [comment]
Interesting idea @MaciejRewera

**Pull request merged**